test/common/testinsp/network.py

```diff
@@ -0,0 +1,52 @@
+from testinsp.base import TestInspector
+from testinsp.constants import YAML, STORE_PATH
+
+
+class NetworkInterfaces(TestInspector):
+    store_type = YAML
+    _get_data_command = "ip -j a"
+
+    def __init__(self, filename=None, pathname=STORE_PATH, external_executor=None):
+        super().__init__(
+            filename=filename, pathname=pathname, external_executor=external_executor
+        )
+        self.exclude_list += ["preferred_life_time", "valid_life_time"]
+
+    def get_data(self):
+        # because of original data are so big for every interface, gather some basic
+        raw_data = self._get_json_from_process(self._get_data_command)
+        data = dict()
+        for item in raw_data:
+            interface = item["ifname"]
+            adresses = list()
+            addrinfo_list = ["local", "prefixlen", "family", "label", "scope"]
+            for item in item["addr_info"]:
+                new_item = dict()
+                for key in addrinfo_list:
+                    new_item[key] = item.get(key)
+                adresses.append(new_item)
+            state = item.get("operstate")
+            data[interface] = {
+                "ifname": interface,
+                "adresses": adresses,
+                "state": state,
+            }
+        return data
+
+
+class FirewallStatus(TestInspector):
+    store_type = YAML
+    _get_data_command = "firewall-cmd --list-all || true"
```

Comment:
Same here, this usually isn't a surprise. If "cockpit" service is not enabled, it's blatantly clear from the test output, and also our test setup explicitly enables it. For other services, the TestFirewall failures show the current state.